server/room.ts

```diff
@@ -41,6 +41,7 @@ export class Room {
   public vBrowser: AssignedVM | undefined = undefined;
   public creationTime: Date = new Date();
   public lock: string | undefined = undefined; // uid of the user who locked the room
+  private isChatEnabled: boolean = true;
```

Comment:
since the default behavior is to enable chat, this should probably be a toggle to disable it. This will also let us use the default state of null as false